parm/metplus/PointStat_UPA.conf

```diff
@@ -242,63 +229,66 @@ OBS_VAR12_LEVELS = L0
 OBS_VAR12_OPTIONS = cnt_thresh = [ >0 ]; cnt_logic = UNION;
 OBS_VAR12_THRESH = >500, >1000, >1500, >2000, >3000, >4000
 
-# End of [config] section and start of [dir] section
+# End of [config] section and start of [dir] section.
 [dir]
-
-# directory containing input to PB2NC
-PB2NC_INPUT_DIR =  {ENV[OBS_DIR]}
-
-# directory to write output from 
-PB2NC_OUTPUT_DIR = {ENV[OUTPUT_BASE]}/metprd/pb2nc
-
-INPUT_BASE = {ENV[INPUT_BASE]}
-
-FCST_POINT_STAT_INPUT_DIR = {INPUT_BASE}
-OBS_POINT_STAT_INPUT_DIR = {PB2NC_OUTPUT_DIR}
-
-# directory containing climatology mean input to PointStat
-# Not used in this example
+#
+# Directory containing observation input to PointStat.
+#
+#OBS_POINT_STAT_INPUT_DIR = {ENV[OUTPUT_BASE]}/metprd/Pb2nc_obs
+OBS_POINT_STAT_INPUT_DIR = {ENV[OBS_INPUT_DIR]}
+#
+# Directory containing forecast input to PointStat.
+#
+FCST_POINT_STAT_INPUT_DIR = {ENV[INPUT_BASE]}
+#
+# Directory containing climatology mean input to PointStat.  Not used in
+# this example.
+#
 POINT_STAT_CLIMO_MEAN_INPUT_DIR =
-
-# directory containing climatology mean input to PointStat
-# Not used in this example
+#
+# Directory containing climatology mean input to PointStat.  Not used in
+# this example.
+#
 POINT_STAT_CLIMO_STDEV_INPUT_DIR =
-
-OUTPUT_BASE = {ENV[OUTPUT_BASE]}
+#
+# Directory in which to write output from PointStat.
+#
 POINT_STAT_OUTPUT_DIR = {OUTPUT_BASE}
+#
+# OUTPUT_BASE apparently has to be set to something; it cannot be left
+# to its default value.  But it is not explicitly used elsewhere in this
+# configuration file.
+#
+OUTPUT_BASE = {ENV[OUTPUT_BASE]}
+#
+# Directory for staging data.
+#
+STAGING_DIR = {OUTPUT_BASE}/stage/UPA
 
-LOG_DIR = {ENV[LOG_DIR]}
-
-STAGING_DIR = {OUTPUT_BASE}/stage/upper_air
-
-# End of [dir] section and start of [filename_templates] section
+# End of [dir] section and start of [filename_templates] section.
 [filename_templates]
-
-# Template to look for prepbvur input to PB2NC relative to PB2NC_INPUT_DIR
-PB2NC_INPUT_TEMPLATE = prepbufr.ndas.{valid?fmt=%Y%m%d%H}
-
-# Template to use to write output from PB2NC
-PB2NC_OUTPUT_TEMPLATE = prepbufr.ndas{ENV[DOT_ENSMEM]}.{valid?fmt=%Y%m%d%H}.nc
-
-# Template to look for forecast input to PointStat relative to FCST_POINT_STAT_INPUT_DIR
+#
+# Template for observation input to PointStat relative to
+# OBS_POINT_STAT_INPUT_DIR.
+#
+#OBS_POINT_STAT_INPUT_TEMPLATE = prepbufr.ndas{ENV[DOT_ENSMEM]}.{valid?fmt=%Y%m%d%H}.nc
```

Comment:
Should this commented out line (274) be removed, or is this going to be coming back in a subsequent PR?

Reply:
Removed, will push soon.